wiki/Game_modifier/en.md

```diff
@@ -14,7 +14,7 @@ tags:
 
 *Not to be confused with [modding](/wiki/Modding).*
 
-![Mod selection screen](img/GM_Main.jpg)
+![Mod selection screen](img/mod-selection-screen.jpg "comparison between the mod selection screens of osu!standard (top-left), osu!taiko (top-right), osu!catch (bottom-left), and osu!mania (bottom-right)")
```

Comment:
it's fairly hard to keep in mind which screenshot is for which mode, because they look the same and have no obvious differences (unlike that 2x2 image of fade out for o!m). please add captions, possibly in every corner that's the closest to the image's center (so, bottom right for standard, bottom left for taiko, top right for catch and top left for mania)

Reply:
That's why I added the tooltip? I'm fairly sure it shows up as actual text on beneath the image. Also, I refrained from adding any text onto the images themselves so that translators don't have to go in and change anything

Reply:
to be honest, the tooltip doesn't help, because you need to constantly return to it, or make a mental effort to memorize it on spot